mobile/src/main/res/layout/dialog_items_unavailable_warning.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I tried just having the `CheckBox` by itself and it stopped respecting the `layout_margin` attribute, so I'm going to leave it in.